Lib/_pyrepl/readline.py

```diff
@@ -161,6 +169,11 @@ def get_completions(self, stem: str) -> list[str]:
             result.sort()
         return result
 
+    def get_module_completions(self) -> list[str]:
+        completer = ModuleCompleter(namespace={'__package__': '_pyrepl'})  # TODO: namespace?
```

Comment:
Either remove the TODO or we should handle this differently

Reply:
I removed the TODO and left a comment. Inside pyrepl, `__package__` is set to `_pyrepl` so the module completer should use the same value when resolving relative packages.